stable/admin/README.md

```diff
@@ -171,6 +171,8 @@ The following tables list the configurable parameters for the `admin` option of
 | `replicas` | Number of NuoDB Admin replicas | `1` |
 | `lbPolicy` | Load balancer policy name | `nil` |
 | `lbQuery` | Load balancer query | `nil` |
+| `k8sL4Service.enabled` | Whether to deploy a Layer 4 cloud load balancer service for the admin layer | `false` |
```

Comment:
I'm not a huge fan of the name including the implementation details.
Sort of like the old "hungarian" notation in Visual C++.

If the `k8sL4` qualification needed to distinguish this option from other `services` for the admin layer?

Reply:
Could we name it "service" or "lbService" or similar?

Reply:
Aarrgh... naming... I had load balancer in there before and Adrian said that's too overloaded and confuses with lb policy / nuodb lb's...

k8sService?
service?
externalService?
... names in a hat :-)

Reply:
Sorry to put you through the "naming from hell" pain...

I would be happy with "externalService" or "externalAccess".
People like BobB prefer names that are consistent with the underlying feature, so I assume some would prefer "externalService" for that reason - but in truth the term "service" is badly overloaded already - so a name relying on it is less helpful.

Reply:
I'm going for externalAccess... :-)